test/functions/diten/param_enum_to_int.good

```diff
@@ -1,2 +1,6 @@
-param 2
-non-param 2
+param_enum_to_int.chpl:13: error: unresolved call 'foo(color)'
```

Comment:
I think I'd rather add explicit casts in this test to make sure that a param enum->int cast still exists.  (It does.)